examples/solvers/bregman_tv_tomography.py

```diff
@@ -0,0 +1,95 @@
+"""Example of Bregman-TV in tomography.
+
+The example does seven iterations in a Bregman-TV scheme, applied to a
+tomography problem.
+"""
+
+import odl
+import numpy as np
+
+# Discrete reconstruction space
+reco_space = odl.uniform_discr(min_pt=[-20, -20], max_pt=[20, 20],
+                               shape=[128, 128], dtype='float32')
+
+# Make a parallel beam geometry with flat detector
+angle_partition = odl.uniform_partition(0, np.pi, 100)
+detector_partition = odl.uniform_partition(-30, 30, 558)
+geometry = odl.tomo.Parallel2dGeometry(angle_partition, detector_partition)
+
+# Create the ray transform
+ray_trafo = odl.tomo.RayTransform(reco_space, geometry)
+
+# Create phantom
+discr_phantom = odl.phantom.shepp_logan(reco_space, modified=True)
+
+# Create sinogram of forward projected phantom with noise
+noise_free_data = ray_trafo(discr_phantom)
+noise = odl.phantom.white_noise(ray_trafo.range)
+noise *= 1 / noise.norm() * noise_free_data.norm() * 0.10
+data = noise_free_data + noise
+
+# Do a FBP-reconstruction
+fbp_op = odl.tomo.fbp_op(ray_trafo, filter_type='Hann', frequency_scaling=0.4)
+fbp_reco = fbp_op(data)
+
+# Display phantom, FBP reconstruction, and sinograms
+discr_phantom.show(title='Phantom')
+fbp_reco.show(title='FBP reconstruction')
+data.show(title='Simulated data (Sinogram)')
+
+# Define components of the variational problem
+reg_param = 0.3
+gradient = odl.Gradient(reco_space)
+
+# l2-squared data matching
+l2_norm = 0.5 * odl.solvers.L2NormSquared(ray_trafo.range).translated(data)
+
+# Isotropic TV-regularization
+l1_norm = odl.solvers.GroupL1Norm(gradient.range)
```

Comment:
Maybe name this `l12_norm` to avoid confusion. Probably other examples have the same issue.